cli/src/pcluster/config/cluster_config.py

```diff
@@ -1374,6 +1374,7 @@ def __init__(
         ssh: HeadNodeSsh = None,
         disable_simultaneous_multithreading: bool = None,
         local_storage: LocalStorage = None,
+        internal_shared_storage_type: str = "ebs",
```

Comment:
Let's create an enum, to avoid typos and be sure the right string "uppercase vs lowercase" will arrive to the cookbook.

Reply:
Done!